### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by ALPANNA-GH**
- add my recent work section
- add about me section
- add message me section
- add footer section
- style for mobile 1st approach 

> Style elements by `grid` and `flexbox` to accurate position for mobile devices according to `Temp2`

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -426,12 +429,12 @@ input[type="submit"]:active {
   justify-content: center;
 }
 
-.itemA ul {
-  margin: auto;
-  padding: 0;
+.item-about ul {
```
Comment (shella12) [inline style.css:432]: - [ ] Please make sure that all your tech category and tech stack are aliged in the center properly.

Figma design | Your design
---|---
Screenshot 2023-07-06 at 7 03 39 PM|Screenshot 2023-07-06 at 7 02 56 PM

File: style.css [hunk 2]
```diff
@@ -331,17 +331,20 @@ input[type="submit"]:active {
...
   margin: 0;
-  width: 90%;
+  width: 100%;
   background-color: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
 }
 
+.bgmask div *:not(.butn, ul) {
+  padding-left: 1rem;
+}
+
```
Comment (shella12) [inline style.css:342]: - [ ] Please add a bit of padding-top in each project card and make sure font-weight is appropriate and matches the provided design guideline.

Figma design |Your design
---|---
Screenshot 2023-07-06 at 6 59 58 PM|Screenshot 2023-07-06 at 7 02 08 PM

File: index.html [hunk 3]
```diff
@@ -81,8 +81,8 @@ <h3>I'm a Software Developer</h3>
   <!-- My Work section -->
   <section class="mywork">
     <div class="item recent-work">
-        <h2 class="maintitle">My Recent Work</h2>
-        <img class="seperator" src="./images/lineseperator.png" />
+      <h2 class="maintitle">My Recent Work</h2>
+      <img class="seperator" src="./images/lineseperator.png" />
     </div>
     <div class="item multi-post">
       <div class="mp-img">
```
Comment (shella12) [inline index.html:88]: - [ ] Kindly adjust the font-weight of your first project title, description and tags as well to match the provided figma design.

Figma| Your's
---|---
Screenshot 2023-07-06 at 7 00 32 PM|Screenshot 2023-07-06 at 7 01 34 PM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @ALPANNA-GH, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great use of semantic tags ✅ 
- Clean code ✅   

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @ALPANNA-GH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**